doc/lightning-cli.1

```diff
@@ -127,7 +127,7 @@ pretty printing of results isn’t pretty\.
 
 .SH AUTHOR
 
-Rusty Russell \fI<rusty@rustcorp.com.au\fR> is mainly to blame\.
+Rusty Russell <\fIrusty@rustcorp.com.au\fR> is mainly to blame\.
```

Comment:
mh the file without `.md` should be autogenerated from the `make doc-all` command, and here I'm missing the change in the `md `file